api/v01/vrp.rb

```diff
@@ -83,8 +83,8 @@ def self.vrp_request_indice_range(this)
       end
 
       def self.vrp_request_date_range(this)
-        this.optional(:start, type: Date, desc: 'Beginning of the range in date format : .') # date format n'est donnable que si on crée vrp au meme endroit que là oùu on résoud, non ?
-        this.optional(:end, type: Date, desc: 'End of the range in date format : .') # date format n'est donnable que si on crée vrp au meme endroit que là oùu on résoud, non ?
+        this.optional(:start, type: Date, desc: 'Beginning of the range in date format : .')
+        this.optional(:end, type: Date, desc: 'End of the range in date format : .')
```

Comment:
```suggestion
        this.optional(:start, type: Date, desc: 'Beginning of the range in date format.')
        this.optional(:end, type: Date, desc: 'End of the range in date format.')
```

typo ?

Also, I do not know how it looks in the doc but, I believe we don't need to end the descriptions with a dot. @fonsecadeline @braktar  do you know what's the best practice? Don't forget that we have descriptions that are multiple sentences, so we have to have a dot inside :/

Reply:
I do not know the best pratice but that is indeed a question I ask myself everytime...
@braktar do you have an idea why we have those '**:**' ?

Reply:
I probably wanted to add the expected date format, but never did.

Examples of swagger-codegen (project we use to generate the doc) never put a dot at the end of a description. Except for description made of multiple sentences (see details of /pet/findByTags resource) https://petstore.swagger.io/#/

Reply:
Thanks for this best pratice sharing !